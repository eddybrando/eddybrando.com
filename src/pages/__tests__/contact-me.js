import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import ContactMePage from '../contact-me';

describe('ContactMePage', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<ContactMePage />);
    expect(result).toMatchSnapshot();
  });
});
