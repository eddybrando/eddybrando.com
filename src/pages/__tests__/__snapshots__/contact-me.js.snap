// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactMePage renders correctly 1`] = `
<Layout
  headerTitle={
    Object {
      "append": " me",
      "highlight": "Contact",
    }
  }
  mainContainerClass="main-container--centered"
  nav={
    Array [
      Object {
        "label": "Home",
        "link": "/",
      },
      Object {
        "label": "My work",
        "link": "/my-work",
      },
    ]
  }
>
  <SEO
    description="Get in touch with me by using one of this awesome platforms! (It's free!)"
    lang="en"
    meta={Array []}
    noTitleTemplate={false}
    title="Contact me"
  />
  <div>
    <p>
      You can always send me a message over
       
      <a
        className="link-block"
        href="https://www.linkedin.com/in/eddybrandovasquez/"
        rel="noopener noreferrer"
        target="_blank"
      >
        LinkedIn
      </a>
      . I do my best to try and answer ASAP.
    </p>
  </div>
</Layout>
`;
